python/ngen_cal/src/ngen/cal/ngen.py

```diff
@@ -176,7 +178,7 @@ def set_defaults(cls, values: Dict):
 
         custom_args = False
         if( args is None ):
-            args = '{} "all" {} "all" {}'.format(catchments, nexus, realization)
+            args = '{} "all" {} "all" {}'.format(catchments.resolve(), nexus.resolve(), realization.name)
```

Comment:
This should either be `realization` as it was or `realization.resolve()` also, or something else? This config in the yaml now fails for me, because the path part gets stripped:
```
    realization: ./Input/realization.json
```
becomes 
```
../ngen /abs/path/to/Input/spatial/catchment_data.geojson "all" /abs/path/to/Input/spatial/nexus_data.geojson "all" realization.json
```

Reply:
The realization *should* get written to the worker sub directory...

Reply:
Ah... okay right... I can see what probably happened here, the run failed for some other reason and when I tried to run the failed command directly I wasn't in the worker directory and assumed this was the reason. I'll revisit.